tap_gitlab/streams/projects.py

```diff
@@ -0,0 +1,13 @@
+from typing import Dict, Any
+from singer import get_bookmark, get_logger
+from tap_gitlab.streams.abstracts import IncrementalStream
+
+LOGGER = get_logger()
+
+class Projects(IncrementalStream):
+    tap_stream_id = "projects"
+    key_properties = ["id"]
+    replication_method = "INCREMENTAL"
+    replication_keys = ["last_activity_at"]  # Use this instead of "updated_at"
```

Comment:
As per documentation, `last_activity_at` is updated based on project activity and project events. `updated_at` is updated whenever the project record is changed in the database.

Could you please revisit choice of replication key here `last_activity_at` and `updated_at`?